docs/marionette.functions.md

```diff
@@ -460,3 +461,17 @@ myView.do('action:two');
 ```
 
 [Live example](https://jsfiddle.net/marionettejs/zzjhm4p1/)
+
+## Marionette.noConflict
+
+Allows you to run multiple instances of Marionette in the same application. After loading the new version, call `noConflict()` to get a reference to it. At the same time the old version will be returned to Backbone.Marionette.
+
+```javascript
+var Mn = require('backbone.marionette');
+
+var MnV3 = require('backbone.marionette/v3');
+
+// Creates non-conflicting reference to secondary Marionette instance
+MnNew = MnNew.noConflict()
```

Comment:
Yes it should be - nice catch. Last-minute rename on my end. Updates incoming...